app/package.json

```diff
@@ -2,13 +2,21 @@
   "name": "@klimadao/app",
   "private": true,
   "scripts": {
-    "dev": "next dev -p 3001",
+    "dev": "npm run lingui:compile && next dev -p 3001",
     "build": "next build",
     "start": "next start",
     "export": "next export",
-    "lint": "next lint"
+    "lint": "next lint",
+    "lingui:extract": "NODE_ENV=test lingui extract",
+    "lingui:compile": "NODE_ENV=test lingui compile",
+    "postinstall" : "npm run lingui:compile"
   },
   "dependencies": {
+    "@babel/core": "^7.16.0",
+    "@lingui/cli": "3.0.3",
+    "@lingui/macro": "3.0.3",
+    "@lingui/react": "3.0.3",
+    "typescript": "4.0.5",
```

Comment:
`typescript` and `@lingui/cli` should be dev-dependencies

I believe `@lingui/macro` should be as well

Reply:
I like to compile translations as part of the build process. But i am okay with commiting js files too.